modules/videopress/shortcode.php

```diff
@@ -158,8 +158,10 @@ function videopress_handle_editor_view_js() {
 		'modal_labels'      => array(
 			'title'     => __( 'VideoPress Shortcode', 'jetpack' ),
 			'guid'      => __( 'Video ID', 'jetpack' ),
-			'w'         => __( 'Video width (in pixels)', 'jetpack' ),
-			'at'        => __( 'Start video after (in seconds)', 'jetpack' ),
+			'w'         => __( 'Video Width', 'jetpack' ),
+			'w_unit'    => __( 'pixels', 'jetpack' ),
+			'at'        => __( 'Start Video After', 'jetpack' ),
```

Comment:
With `seconds` split off into a new string, this one should have more context for translators.